test-infra/pom.xml

```diff
@@ -66,9 +66,15 @@
         <configuration>
           <instructions>
             <Bundle-SymbolicName>com.datastax.oss.driver.testinfra</Bundle-SymbolicName>
+            <!-- allow SessionUtils to instantiate sessions by reflection -->
+            <DynamicImport-Package>*</DynamicImport-Package>
             <!-- TODO: Only needed to use TestConfigLoader in integration-tests, remove this when we have
```

Comment:
I confess I didn't fully understand this TODO so I left it intact.

Reply:
It's not needed any more now that #1032 is merged and `TestConfigLoader` has been deleted.